src/fileutil.c

```diff
@@ -63,6 +63,15 @@ rs_file_open(char const *filename, char const *mode)
 	    return stdin;
     }
 
+    if (!force && is_write) {
+	if ((f = fopen(filename, "rb"))) {
+	    // File exists
+	    rs_error("File exists \"%s\", aborting!", filename);
+	    fclose(f);
+	    exit(RS_IO_ERROR);
```

Comment:
Wow, that's annoying I didn't make this return a status.  Still, probably better to return NULL than to exit the whole program.